modules/reitit-core/src/reitit/impl.cljc

```diff
@@ -155,3 +157,53 @@
 
 (defn strip-nils [m]
   (->> m (remove (comp nil? second)) (into {})))
+
+;;
+;; Path-parameters, see https://github.com/metosin/reitit/issues/75
```

Comment:
The issue still doesn't explain what this is about, there's only discussion about a possible implementation. It'd be better to have a short explanation here.

Reply:
added